l10n_it_website_sale_fiscalcode/views/templates.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Nicola Malcontenti - Agile Business Group
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+<template id="l10n_it_website_sale_fiscalcode" inherit_id="website_sale.checkout">
+  <xpath expr="//t[@t-if='not only_services']" position="after">
+    <div class="row" id="fiscalcode">
+      <label for="fiscalcode" style="font-weight: normal">Fiscal Code</label>
+      <input type="text" name="fiscalcode" class="form-control" t-value="request.env['res.users'].browse(request.uid).partner_ids.fiscalcode"/>
```

Comment:
Wouldn't be `partner_id.fiscalcode` instead of `partner_ids.fiscalcode`?

Reply:
i don't know how this mess was possible.... Fixed